extensions/widgets/svgpath/svgpath.lcb

```diff
@@ -107,6 +107,9 @@ metadata iconPresetName.editor is "com.livecode.pi.svgicon"
 metadata iconPresetName.default is "Star"
 metadata iconPresetName.label is "Preset"
 
+property iconName get mPathName
+metadata iconName.label is "IconName"
+
```

Comment:
Thanks @BerndN 

I think the label should be "Icon name" instead of "IconName", so as if the user chooses `Property Labels -> Description of option` to see the actual description of the property.

BTW I just noticed that the properties `scaledWidth` and `scaledHeight` are affected by the same issue, i.e. they need to have a label with the descriptive name. But this should not be part of this PR. I'll file a new report so that we can track this separately.

Reply:
Not to mention "IconPresetName" which is probably a misnomer and a funny word. No name is visible...

Reply:
fixed label to "Icon Name"